### PR TITLE
Update docs on Docker's registry mirrors

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Add 'config docker accelerator detail' in get-started

### CODE REVIEW

**Comment by darionyaphet** [comment]
LGTM

File: docs/get-started.md [hunk 1]
```diff
@@ -28,8 +28,20 @@ docker pull vesoft/nebula-graph:latest
 to get `nebula` image and `docker images` that can display images status.
 
 If `docker` is slow when pulling the image, configure an accelerator.
```
Comment (dutor) [inline docs/get-started.md]: "If it's too slow to pull the Docker image, you could configure your own preferred registry mirrors".

I don't know whether the word **accelerator** is a convention in the Docker world or not, but I think you are inventing one. Usually, it's called mirrors.

Besides, Docker, as a product, just like Github and Nebula Graph, we usually don't style them monospace.

File: docs/get-started.md [hunk 2]
```diff
@@ -28,8 +28,20 @@ docker pull vesoft/nebula-graph:latest
...
+1. Check if the `/etc/docker` folder exists, if not create a new one with `mkdir -p /etc/docker`. Generally the folder will exist after docker installation.
```
Comment (dutor) [inline docs/get-started.md]: Is there a comma needed after _if not_? And a _the_ before `docker installation`?

Comment (Amber1990Zhang) [reply docs/get-started.md]: Yes, a comma is better but I don't think ' a the before `docker installation`' is needed.

File: docs/get-started.md (hunk 1 above)
Comment (Amber1990Zhang) [reply docs/get-started.md]: fixed

File: docs/get-started.md [hunk 3]
```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
 
 ### Step 1 Install Nebula Graph
 
-The easiest way to startup `nebula` is using `docker`.
+The easiest way to startup `Nebula` is using `Docker`.
```
Comment (dutor) [inline docs/get-started.md]: no backtick around **Docker** and **Nebula**

File: docs/get-started.md [hunk 4]
```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
...
 
-First of all, you should make sure that `docker` has been installed on your machine. Open a terminal and run the following command :
+First of all, you should make sure that `Docker` has been installed on your machine. Open a terminal and run the following command :
 
 ```
 docker --version
 ```
 
-If `docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install docker.
+If `Docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install Docker.
```
Comment (dutor) [inline docs/get-started.md]: ditto

File: docs/get-started.md [hunk 5]
```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
...
 
-The easiest way to startup `nebula` is using `docker`.
+The easiest way to startup `Nebula` is using `Docker`.
 `Docker` is a tool designed to make it easier to create, deploy, and run applications by using containers.
 Containers allow a developer to package up an application with all of the parts it needs,
 such as libraries and other dependencies, and ship it all out as one package.
 By doing so, the developer can rest assured that the application will run on any other `Linux` machine regardless of any customized settings that machine might have that could differ from the machine used for writing and testing the code.
 
-First of all, you should make sure that `docker` has been installed on your machine. Open a terminal and run the following command :
+First of all, you should make sure that `Docker` has been installed on your machine. Open a terminal and run the following command :
```
Comment (dutor) [inline docs/get-started.md]: ditto

File: docs/get-started.md [hunk 6]
```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
 
 ### Step 1 Install Nebula Graph
 
-The easiest way to startup `nebula` is using `docker`.
+The easiest way to startup `Nebula` is using `Docker`.
 `Docker` is a tool designed to make it easier to create, deploy, and run applications by using containers.
```
Comment (dutor) [inline docs/get-started.md]: ditto

File: docs/get-started.md [hunk 7]
```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
...
 
-If `docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install docker.
+If `Docker` is not found, please see [here](https://docs.docker.com/install/) for more information to install Docker.
 
 After that, using
 ```
 docker pull vesoft/nebula-graph:latest
 ```
-to get `nebula` image and `docker images` that can display images status.
+to get `Nebula` image and `Docker images` that can display images status.
```
Comment (dutor) [inline docs/get-started.md]: ditto. 
Besides, here the `docker` is a command, so it should be in backticks and cannot be capitalized.

File: docs/get-started.md [hunk 8]
```diff
@@ -7,29 +7,41 @@ This tutorial provides a quick introduction to use `Nebula Graph`.
...
 
-If `docker` is slow when pulling the image, configure an accelerator.
+If `Docker` is slow when pulling the image, configure a new mirror.
```
Comment (dutor) [inline docs/get-started.md]: Ditto.

Besides, what is slow is the very pulling action itself, but not Docker.

File: docs/get-started.md (hunk 3 above)
Comment (Amber1990Zhang) [reply docs/get-started.md]: Thanks for your patience review and professional suggestions, I've learned a lot. Have fixed all the backticks and expressions.

File: docs/get-started.md (hunk 4 above)
Comment (Amber1990Zhang) [reply docs/get-started.md]: Thanks for your patient review and professional suggestions, I've learned a lot. Have fixed all the backtick and expression.

**Comment by dutor** [comment]
Jenkins, go!

**Comment by laura-ding** [comment]
jenkins go

**Pull request merged**